From: Dmitry Borodaenko <angdraug@debian.org>
Date: Sun, 8 Jan 2012 16:00:44 +0300
Subject: skip safe mode test

This test produces a false negative when run under /tmp, making it
impossible to build this package with cowbuilder and the like.
---
 spec/fast_gettext/translation_repository/mo_spec.rb | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/spec/fast_gettext/translation_repository/mo_spec.rb b/spec/fast_gettext/translation_repository/mo_spec.rb
index baca690..85135db 100644
--- a/spec/fast_gettext/translation_repository/mo_spec.rb
+++ b/spec/fast_gettext/translation_repository/mo_spec.rb
@@ -49,10 +49,4 @@ describe 'FastGettext::TranslationRepository::Mo' do
     rep['car'].should == 'Test'#just check it is loaded correctly
     rep.pluralisation_rule.call(2).should == 3
   end
-
-  it "can work in SAFE mode" do
-    pending_if RUBY_VERSION > "2.0" do
-      `ruby spec/cases/safe_mode_can_handle_locales.rb 2>&1`.should == 'true'
-    end
-  end
 end
